js/src/components/google-combo-account-card/connected-google-combo-account-card.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * External dependencies
+ */
+import { createInterpolateElement } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+import { Spinner } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import AccountCard, { APPEARANCE } from '../account-card';
+import useGoogleAdsAccount from '.~/hooks/useGoogleAdsAccount';
+import useGoogleMCAccount from '.~/hooks/useGoogleMCAccount';
+import AppSpinner from '../app-spinner';
+import useCreateAccounts from '../../hooks/useCreateAccounts';
+
+/**
+ * Clicking on the "connect to a different Google account" button.
+ *
+ * @event gla_google_account_connect_different_account_button_click
+ */
+
+/**
+ * Renders a Google account card UI with connected account information.
+ * It will also kickoff Ads and Merchant Center account creation if the user does not have accounts.
+ *
+ * @param {Object} props React props.
+ * @param {{ googleAccount: object }} props.googleAccount The Google account.
+ *
+ * @fires gla_google_account_connect_different_account_button_click
+ */
+const ConnectedGoogleComboAccountCard = ( { googleAccount } ) => {
+	const {
+		googleMCAccount,
+		hasFinishedResolution: hasFinishedResolutionForCurrentMCAccount,
+	} = useGoogleMCAccount();
+
+	const {
+		googleAdsAccount,
+		hasFinishedResolution: hasFinishedResolutionForCurrentAdsAccount,
+	} = useGoogleAdsAccount();
+
+	const {
+		accountsCreated,
+		accountCreationChecksResolved,
+		isCreatingAccounts,
+	} = useCreateAccounts();
+
+	if (
+		! accountsCreated &&
+		( ! hasFinishedResolutionForCurrentAdsAccount ||
+			! hasFinishedResolutionForCurrentMCAccount ||
+			! accountCreationChecksResolved )
+	) {
+		return <AccountCard description={ <AppSpinner /> } />;
+	}
+
+	const creatingAccounts =
+		isCreatingAccounts ||
+		( accountsCreated &&
+			( ! hasFinishedResolutionForCurrentMCAccount ||
+				! hasFinishedResolutionForCurrentAdsAccount ) );
+
+	const Description = () => {
+		if ( creatingAccounts ) {
+			return createInterpolateElement(
+				__(
+					'<p>You don’t have Merchant Center nor Google Ads accounts, so we’re creating them for you.</p>',
+					'google-listings-and-ads'
+				),
+				{
+					p: <p></p>,
+				}
+			);
+		}
+
+		return (
+			<>
```

Comment:
@ankitrox Let's wrap the paragraphs within a `div` with a class name.
```suggestion
			<div className="gla-connected-google-combo-account-card__description">
```

and then add the following CSS:
```scss
.gla-connected-google-combo-account-card__description p {
    margin: 0;
}
```

Thus we won't need to set `gap: 0` for `.gla-account-card__description`

Reply:
Done.